pulsar-common/pom.xml

```diff
@@ -101,6 +101,16 @@
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.bookkeeper</groupId>
+      <artifactId>bookkeeper-common</artifactId>
+    </dependency>
+
+    <dependency>
```

Comment:
@sijie Is it ok to introduce the bookkeeper-server into the pulsar-common?

Reply:
Why do you need to introduce the bookkeeper-server module? We just need `EnsemblePlacementPolicyConfig`, no? This class just need a class and a map of properties.

Reply:
Removed the bookkeeper-server.